src/awkward/operations/ak_to_feather.py

```diff
@@ -24,13 +24,39 @@ def to_feather(
     compression="zstd",
     compression_level=None,
     chunksize=None,
-    feather_version=2,
+    version=2,
```

Comment:
I think it's good for this to be called `feather_version`, since it would be ambiguous what the "version" is referring to otherwise.